scifight/models.py

```diff
@@ -16,6 +17,16 @@
     hold a number plus possible brief one-word explanation. """
 
 
+class Tournament(models.Model):
+    name        = models.CharField(max_length=150)
+    slug        = models.SlugField(max_length=150, unique=True)
```

Comment:
Why hardcode lengths? Use `NAME_LENGTH` and `TEXT_LENGTH` for `name` and `description`, and introduce `SLUG_LENGTH` for all slugs. We don't have to make the same decisions more than once.
BTW, 150 characters for a slug is an *awful* lot. I'd choose 20 instead.